src/style.css

```
@@ -1 +1,56 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  background-color: #fff;
+  padding: 0 50px;
+  align-items: center;
+}
+
+.nav__item {
+  font-family: "Roboto", sans-serif;
+  list-style: none;
+  font-size: 12px;
+  display: inline-block;
```

Why inline-block? It's unnecessary, you have flex wrapper
```suggestion
  display: block;
```